File: 03_recursion/swift/02_greet.swift
import Foundation

func greet2(name: String) {
    print("how are you, \(name) ?")
}

func bye() {
    print("ok bye!")
}

func greet(name: String ) {
    print("hello, \(name)!")
    greet2(name: name)
    print("getting ready to say bye...")
    bye()
}

greet(name: "adit")

File: 03_recursion/php/02_greet.php
<?php

function greet2($name)
{
    echo 'how are you, ' . $name . '?';
}

function bye()
{
    echo 'ok bye!';
}

function greet($name)
{
    echo 'hello, ' . $name . '!';
    greet2($name);
    echo  'getting ready to say bye...';
    bye();
}

greet('adit');
